@navikt/core/react/src/form/combobox/Input/Input.tsx

```diff
@@ -43,17 +43,18 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
 
     const onEnter = useCallback(
       (event: React.KeyboardEvent) => {
-        const isTextInSelectedOptions = (text: string) => {
-          return selectedOptions.find(
-            (item) => item.toLocaleLowerCase() === text.toLocaleLowerCase(),
+        const isTextInSelectedOptions = (text: string): boolean => {
+          return !!selectedOptions.find(
+            (item) =>
+              item.label.toLocaleLowerCase() === text.toLocaleLowerCase(),
           );
         };
```

Comment:
Her passer det kanskje bedre med `.some()`
```suggestion
        const isTextInSelectedOptions = (text: string) =>
          selectedOptions.some(
            (option) =>
              option.label.toLocaleLowerCase() === text.toLocaleLowerCase(),
          );
```